quickwit/quickwit-search/src/lib.rs

```diff
@@ -266,3 +266,19 @@ pub async fn start_searcher_service(
     ));
     Ok(search_service)
 }
+
+#[doc(hidden)]
```

Comment:
Can we put this macro behind a feature flag? `#[cfg(any(test, feature = "testsuite"))]`